pkg/querier/querier.go

```diff
@@ -264,24 +228,80 @@ func NewQueryable(
 
 // multiQuerier implements storage.Querier, orchestrating requests across a set of queriers.
 type multiQuerier struct {
-	queriers []storage.Querier
-
-	chunkIterFn chunkIteratorFunc
-	ctx         context.Context
-	minT, maxT  int64
+	distributor  storage.Queryable
+	blockStore   storage.Queryable
+	queryMetrics *stats.QueryMetrics
+	cfg          Config
+	chunkIterFn  chunkIteratorFunc
+	minT, maxT   int64
 
 	maxQueryIntoFuture time.Duration
 	limits             *validation.Overrides
 
 	logger log.Logger
 }
 
+func (mq multiQuerier) getQueriers(ctx context.Context) (context.Context, []storage.Querier, error) {
```

Comment:
Nit, but idiomatic goland way is to omit the `get` prefix calling this just `queriers()`